content/03.methods.md

```diff
@@ -48,6 +48,32 @@ We trained this model using scikit-learn's [@raw:scikit-learn] implementation of
 For each principal component we calculated its cosine similarity with  all tokens in our word2vec model's vocabulary.
 We report the top 100 positive and negative scoring tokens in the form of  word clouds, where the size of each word corresponds to the magnitude of similarity and color represents positive (blue) or negative (orange) association.
 
-### Recommending Journals/ bioRxiv Audience Analysis
-1. This title will update as analysis is completed
-2. This section will describe how the above process is conducted
+### Journal Recommendation
+
+We aimed to predict the journal a paper would eventually be published in based on its embedding representation.
+We illustrate our recommendations as a short list along with a network visualization available at [website link here once available](link goes here when live).
+Since our approach is solely focused on embeddings, we utilized the k-nearest neighbors technique with Euclidean distance to recommend journals.

```

Comment:
```suggestion
Since we sought to examine if embeddings were related to publication venue, we used a simple k-nearest neighbors approach with Euclidean distance to recommend journals.
```